docs/coverage/repositories/repository-dashboard.md

```diff
@@ -0,0 +1,79 @@
+---
+product: coverage
+---
+
+# [Coverage] Repository Dashboard
+<!--TODO
+    Review page, see https://codacy.atlassian.net/browse/COV-115 for more details-->
+
+The **Repository Dashboard** provides an overview of the repository coverage and items that require your attention.
+
+To access your Repository Dashboard, select a repository from the [Organization Dashboard](../../organizations/organization-overview.md) or open a repository on any other page and select **Dashboard** on the left navigation sidebar.<!--TODO Review navigation-->
+
+!!! tip
+    You can share the URL of the Repository Dashboard for your **public repositories** to allow other people to see your repository coverage metrics, even if they aren't registered on Codacy.
+
+![Repository Dashboard](images/repository-dashboard.png)<!--TODO Update-->
```

Comment:
This image and the one on the [equivalent Quality page](https://feat-create-coverage-pages-cov-256--docs-codacy.netlify.app/repositories/repository-dashboard/#repository-dashboard) seem inverted here.

Reply:
Fixed, see https://github.com/codacy/docs/pull/1658#discussion_r1164400047! :bow: